contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -29,6 +29,7 @@
 #include "pltsql_instr.h"
 #include "parser/parser.h"
 #include "parser/parse_target.h"
+#include "parser/parse_relation.h"
```

Comment:
why this header?

Reply:
we were using `attnameAttNum() `function without including the header where its declaration has been done.